nxdrive/gui/application.py

```diff
@@ -685,6 +698,27 @@ def show_conflicts_resolution(self, engine: Engine) -> None:
         self._window_root(self.conflicts_window).setEngine.emit(engine.uid)
         self._show_window(self.conflicts_window)
 
+    @pyqtSlot(str)
+    def show_direct_transfer_window(self, engine_uid: str) -> None:
+        """Display the Direct Transfer window."""
+        self._window_root(self.direct_transfer_window).setEngine.emit(engine_uid)
+        screen = self.direct_transfer_window.screen()
+        height = screen.size().height()
+        width = screen.size().width()
+        self.direct_transfer_window.setX(
+            (width / 2) - (self.direct_transfer_window.minimumWidth() / 2)
+        )
+        self.direct_transfer_window.setY(
+            (height / 2) - (self.direct_transfer_window.minimumHeight() / 2)
+        )
```

Comment:
- Add blank lines before and after the code block.
- Add a comment like "# Center the window on the screen"